packages/Modal/src/components/Footer.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import { spacer } from "@paprika/stylers/lib/helpers";
+import tokens from "@paprika/tokens";
+
+const Footer = styled.div`
+  align-items: center;
+  background-color: ${tokens.modal.footer.backgroundColor};
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+  padding: ${spacer(1.5)};
+
+  & > * {
+    margin: ${spacer(0.5)};
```

Comment:
padding: ${tokens.spaceSm};

Reply:
On second though, this seems a bit intrusive 🤔 
Again I think the parent container should be agnostic of its contents.

Reply:
@nahumzs @mikrotron What do we think if we wrap each child to `div` an assign styles to this `div`? So we can avoid problems if children have its own margins. They will never overridden.

https://github.com/acl-services/paprika/pull/320/commits/feae3ecbe58dce863f1b203b1b4e9afeffae82af
https://github.com/acl-services/paprika/pull/320/commits/6989fbc9bfb30fde54b4c576a117d4ab31bcb245

Reply:
I think we can simplify the `<Modal.Footer>` and render more as a pure container.  In some cases we might want to include a "do not show again" checkbox or something similar, not sure if this will handle that?  It does allow for a convenient API, however...

Soon we will have a new component, `<DialogActions>` that will control the layout of the usual footer (two or three buttons).  It is in progress, but you may have some suggestions for it:
https://github.com/acl-services/paprika/pull/305

What do you think?

Reply:
IMO, `<DialogActions>` does nothing. My suggestion:
1. Don't have hardcoded set of buttons. Buttons should be defined on consumer side.
2. `<Modal.Footer>` should be responsible for positioning of buttons.
3. `In some cases we might want to include a "do not show again" checkbox or something similar` For this case I suggest to add `disablePositioning` to `<Modal.Footer>`. When this props passed, `<Modal.Footer>` disable all style rules except `background-color` and `padding`.

In other words, I suggest to move all responsibilities of `<DialogActions>` to `<Modal.Footer>`, but do not hardcode set of buttons to it.

@mikrotron WDYT?

Reply:
The original `<DialogActions>` was created as a way to share the `<Modal.Footer>` in acl-ui with `<SidePanel>` and other "dialogs".  If we continue with this new implementation of the `<Modal.Footer>`, we will likely find ourselves in the same situation, where we want to extract it to share it.  Neither implementations do much, besides control the layout of buttons (a deliberate design choice that has been debated and committed to by the UX designers – but also subject to change) as well as a bit of styling for the spacing.  

I think I prefer for the new `<Modal.Footer>` to be a fairly basic container (with a bit of styling for background color, for instance) and the `<DialogActions>` to contain the styling / layout for the buttons we will use 95% of the time. 

That said, since the `<DialogActions>` isn't ready yet, we can merge this for now and refactor a little later (including the `<SidePanel>`).

@nahumzs ?

Reply:
Sorry, little late response, but we already agree that all the footer header and body/content should be agnostic of whatever is render on them, pretty much the only thing they do is providing padding/margins/background as necessary if they are use it. 

for now, let's merge it, but let's agree that any area that will display content in our modals shouldn't affect or mess with it's content unless the consumer uses Modal.Footer or Modal.Header, Sidepanel.Header, Takeover.Footer etc. 

If you don't use neither of Header, Footer or Content, the consumer "content" should expand as much as possible under our modals. 

cc @mikrotron